core/src/test/resources/struts.xml

```diff
@@ -20,8 +20,8 @@
  */
 -->
 <!DOCTYPE struts PUBLIC
-          "-//Apache Software Foundation//DTD Struts Configuration 2.6//EN"
-          "http://struts.apache.org/dtds/struts-2.6.dtd">
+        "-//Apache Software Foundation//DTD Struts Configuration 6.0//EN"
+        "struts-6.0.dtd">
```

Comment:
Isn't it possible to provide the dtd under https://struts.apache.org/dtds/ anymore?

Reply:
It is, just forgot to add it here https://github.com/apache/struts-site/tree/master/source/dtds